src/AccelerationPlanner.cpp

```diff
@@ -181,14 +181,29 @@ bool AccelerationPlanner::appendVelocityQueue(TwistStampedArray& current_twists,
 }
 
 // Receive a new list of velocities commands and call appendVelocityQueue to insert them into the queue
-// TODO: check that message is sorted properly, default to safe state if anythings wrong?
+// The messages in the array should arrive in chronological order (from message[0] to message[n])
 void AccelerationPlanner::processVelocityCommand(const iarc7_msgs::TwistStampedArray::ConstPtr& message)
 {
 
+    TwistStampedArray::const_iterator checkMessageOrder;
+
+    for(checkMessageOrder = message->twists.begin(); checkMessageOrder != message->twists.end() - 1; 
```

Comment:
I think a more readable style is to put each of these statements on their own line. Instead of 2 on the top and one on the bottom.